.circleci/config.yml

```diff
@@ -140,6 +140,17 @@ jobs:
           name: Check nonce overrides
           command: bash ./script/utils/check-nonce-overrides.sh
 
+  simulate_non_terminal_tasks:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Simulate non-terminal tasks
+          command: bash ./script/utils/simulate-tasks.sh
```

Comment:
It might be nice to have this run but always pass for the time being. The idea would be to catch edge cases before allowing it to block people if it fails.